src/screens/augmentedReality/ARShowScreen.js

```diff
@@ -141,10 +194,12 @@ const objectParser = async ({ item, setObject, setIsLoading, onPress }) => {
         temperature: item?.temperature,
         uri: item?.uri
       };
+    } else {
+      return;
```

Comment:
what is this return useful for?

Reply:
I guess it's not doing anything. I'm already checking above that the type is a texture.